runtime/lib/app-runtime.js

```diff
@@ -1117,16 +1139,10 @@ AppRuntime.prototype.multimediaMethod = function (name, args) {
  * @private
  */
 AppRuntime.prototype.reconnect = function () {
-  wifi.resetDns()
```

Comment:
Why removing this?

Reply:
Dns state has automatically handled in net_manager.

Reply:
AFAIK `wifi.resetDns` resolves an issue related to dns issue in client processes that after network switching no domain could be resolved unless restarting the process or flushing dns. And as the issue is introduced with gcc so that it has to be handled in client processes, but nothing todo with net_manager.